src/services/import/markdown.ts

```diff
@@ -0,0 +1,17 @@
+"use strict";
+
+import marked = require("marked");
+import htmlSanitizer = require('../html_sanitizer');
+import importUtils = require('./utils');
+
+function renderToHtml(content: string, title: string) {
+    const html = marked.parse(content, {
+        async: false
+    }) as string;  // FIXME: mangle and headerIds does not seem to exist in marked
```

Comment:
@j-tirelli , after some digging I was able to identify them in the documentation of `marked` for [options](https://marked.js.org/using_advanced#options).

`mangle` was set by default to `true` and now says:

> Removed in v8.0.0 use [marked-mangle](https://www.npmjs.com/package/marked-mangle) to mangle email addresses.

`headerIds` was also set by default to `true`:

> Removed in v8.0.0 use [marked-gfm-heading-id](https://www.npmjs.com/package/marked-gfm-heading-id) to include an id attribute when emitting headings (h1, h2, h3, etc).

zadam added `marked` back in [July 2023](https://github.com/zadam/trilium/commit/64b86b2), but with a pretty old version for some reason (5.1.1). Since then it was updated multiple times, so probably it just ended up as technical debt.

Since we now know what's the reason behind them, we can safely remove the TODO.